deepchem/models/tests/test_layer_graphnet.py

```diff
@@ -0,0 +1,32 @@
+import deepchem as dc
+import pytest
+try:
+  import torch
+  import deepchem.models.torch_models.layers as torch_layers
+  has_torch = True
+except ModuleNotFoundError:
+  has_torch = False
+
+
+@pytest.mark.torch
+def test_graphnet_layer():
```

Comment:
We should add another test with fixed (not random) input that tests that the layer output is correct (matches a specified fixed output)